<launch>
    <!-- global configuration -->
    <rosparam command="load" file="$(find genesis_parking)/config/parking.yaml" />

    <!-- state estimate -->
    <node pkg="genesis_parking" type="state_publisher.py" name="state_publisher" output="screen" />

    <!-- visualization -->
    <node pkg="genesis_parking" type="simulator.py" name="parking_viz" output="screen" />    
    
</launch>
